src/charm.py

```diff
@@ -190,12 +217,15 @@ def _on_config_changed(self, _: ConfigChangedEvent) -> None:
 
         self.reconfigure_scrape_target()
         self.reconfigure_open_ports()
-        self.unit.status = ActiveStatus("Unit is ready")
 
     def _on_prometheus_available(self, _: PrometheusConnected) -> None:
         """Trigger configuration of a prometheus scrape target."""
         self.reconfigure_scrape_target()
 
+    @evaluate_status
+    def _on_update_status(self, _: UpdateStatusEvent) -> None:
+        """Asses unit's status."""
```

Comment:
Typo: Asses -> Assess

Reply:
:laughing: best typo ever. Thanks for spotting it.